### PR TITLE
Added 5 Beers. More to follow...

### PR DESCRIPTION
**Pull request by AliSkinner**
Saw your Tweet about this project. Great idea. Happy to add a bunch more...

In this commit:
Blitz Berliner Weisse, Dead Metaphor, Dogma, Hello My Name is Sonja, Shopwrecker Circus

### CODE REVIEW

**Comment by sammdec** [comment]
Oh wow thanks so much :+1: Any contributions are greatly received.

File: data/blitz_berliner_weisse.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+      }
+    ],
+    "hops": [
+      {
+        "name": "Magnum",
+        "amount": {
+          "value": 8.5,
+          "unit": "grams"
+        },
+        "add": "Start",
```
Comment (sammdec) [inline data/blitz_berliner_weisse.json]: Im trying to keep all these to lowercase, would you mind updating

File: data/blitz_berliner_weisse.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+      "duration": 60
+    },
+    "fermentation": {
+      "temp": {
+        "value": 19,
+        "unit": "celsius"
+      }
+    },
+    "twist": {
+      "technique": "Before main fermentation steep a bag of malt in the wort for 48 hours. Re boil the wort before pitching in the yeast for the main fermentation"
```
Comment (sammdec) [inline data/blitz_berliner_weisse.json]: I have now removed tehcnique and instead flattened it to just a string value for `twist`, would you be cool to do the same

File: data/blitz_berliner_weisse.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+        "name": "Magnum",
+        "amount": {
+          "value": 8.5,
+          "unit": "grams"
+        },
+        "add": "Start",
+        "attribute": "Bitter"
+      }
+    ],
+    "yeast": "Wyeast 1056 - American Ale IITM"
```
Comment (sammdec) [inline data/blitz_berliner_weisse.json]: Could you use the proper ™ symbol

File: data/dead_metaphor.json [hunk 4]
```diff
@@ -0,0 +1,144 @@
+{
+  "name": "Dead Metaphor",
+  "tagline": "Scottish Breakfast Stout",
```
Comment (sammdec) [inline data/dead_metaphor.json]: Could you add a full stop to the end please :smile:

File: data/dead_metaphor.json [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+      },
+      "duration": 75
+    },
+    "fermentation": {
+      "temp": {
+        "value": 19,
+        "unit": "celsius"
+      }
+    },
+    "twist": {
```
Comment (sammdec) [inline data/dead_metaphor.json]: Would you mind flattening this to a string I have just been doing it like this

```
"twist": "Cacao: 37.5g at end, Lactose: 125g at end, Ground Coffee: 6.3g at end"
```

The reason being is that as you may find, alot of recipes with twists don't follow a pattern so I figure a string is the safest option

File: data/dead_metaphor.json [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+      }
+    ],
+    "hops": [
+      {
+        "name": "Magnum",
+        "amount": {
+          "value": 25,
+          "unit": "grams"
+        },
+        "add": "Start",
```
Comment (sammdec) [inline data/dead_metaphor.json]: Again lowercase if thats cool :+1:

File: data/dead_metaphor.json [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+        "attribute": "Bitter"
+      }
+    ],
+    "yeast": "Wyeast 1056 - American AleTM"
```
Comment (sammdec) [inline data/dead_metaphor.json]: ™ symbol if that's ok its `alt + shift + 2` on a mac :+1:

File: data/dogma.json [hunk 8]
```diff
@@ -0,0 +1,157 @@
+{
+  "name": "Dogma",
+  "tagline": "Revamped wee heavy. Luscious. Malty. Fruity",
```
Comment (sammdec) [inline data/dogma.json]: Full stop at end please

File: data/dogma.json [hunk 9]
```diff
@@ -0,0 +1,157 @@
...
+      },
+      "duration": 75
+    },
+    "fermentation": {
+      "temp": {
+        "value": 12,
+        "unit": "celsius"
+      }
+    },
+    "twist": {
```
Comment (sammdec) [inline data/dogma.json]: Again I think just a string, cheers

File: data/dogma.json [hunk 10]
```diff
@@ -0,0 +1,157 @@
...
+      }
+    ],
+    "hops": [
+      {
+        "name": "Saaz",
+        "amount": {
+          "value": 75,
+          "unit": "grams"
+        },
+        "add": "Start",
```
Comment (sammdec) [inline data/dogma.json]: lowercase

File: data/dogma.json [hunk 11]
```diff
@@ -0,0 +1,157 @@
...
+        "attribute": "Bitter"
```
Comment (sammdec) [inline data/dogma.json]: lowercase

File: data/dogma.json [hunk 12]
```diff
@@ -0,0 +1,157 @@
...
+        "name": "First Gold",
+        "amount": {
+          "value": 10,
+          "unit": "grams"
+        },
+        "add": "Start",
+        "attribute": "Bitter"
+      }
+    ],
+    "yeast": "Wyeast 2007 - Pilsen LagerTM"
```
Comment (sammdec) [inline data/dogma.json]: ™

File: data/hello_my_name_is_sonja.json [hunk 13]
```diff
@@ -0,0 +1,117 @@
+{
+  "name": "Hello My Name is Sonja (w/Evil Twin)",
+  "tagline": "Blueberry Double IPA",
```
Comment (sammdec) [inline data/hello_my_name_is_sonja.json]: fullstop

File: data/hello_my_name_is_sonja.json [hunk 14]
```diff
@@ -0,0 +1,117 @@
...
+      }
+    ],
+    "hops": [
+      {
+        "name": "Mosaic",
+        "amount": {
+          "value": 12.5,
+          "unit": "grams"
+        },
+        "add": "Start",
```
Comment (sammdec) [inline data/hello_my_name_is_sonja.json]: lowercase

File: data/hello_my_name_is_sonja.json [hunk 15]
```diff
@@ -0,0 +1,117 @@
...
+        "name": "Mosaic",
+        "amount": {
+          "value": 187.5,
+          "unit": "grams"
+        },
+        "add": "Dry Hop",
+        "attribute": "Aroma"
+      }
+    ],
+    "yeast": "Wyeast 1272 - American Ale IITM"
```
Comment (sammdec) [inline data/hello_my_name_is_sonja.json]: ™

File: data/shopwrecker_circus.json [hunk 16]
```diff
@@ -0,0 +1,169 @@
+{
+  "name": "Shopwrecker Circus (w/Oskar Blues)",
```
Comment (sammdec) [inline data/shopwrecker_circus.json]: Think this is Shipwrecker?

File: data/shopwrecker_circus.json [hunk 17]
```diff
@@ -0,0 +1,169 @@
...
+  "tagline": "American Barley Wine - Oskar Blues Collab",
```
Comment (sammdec) [inline data/shopwrecker_circus.json]: fullstop

File: data/shopwrecker_circus.json [hunk 18]
```diff
@@ -0,0 +1,169 @@
...
+        "unit": "celsius"
+      },
+      "duration": 75
+    },
+    "fermentation": {
+      "temp": {
+        "value": 21,
+        "unit": "celsius"
+      }
+    }
```
Comment (sammdec) [inline data/shopwrecker_circus.json]: could you leave in "twist" and set it to null

File: data/shopwrecker_circus.json [hunk 19]
```diff
@@ -0,0 +1,169 @@
...
+      }
+    ],
+    "hops": [
+      {
+        "name": "Chinook",
+        "amount": {
+          "value": 50,
+          "unit": "grams"
+        },
+        "add": "Start",
```
Comment (sammdec) [inline data/shopwrecker_circus.json]: lowercase

File: data/shopwrecker_circus.json [hunk 20]
```diff
@@ -0,0 +1,169 @@
...
+        "name": "Citra",
+        "amount": {
+          "value": 125,
+          "unit": "grams"
+        },
+        "add": "Dry Hop",
+        "attribute": "Aroma"
+      }
+    ],
+    "yeast": "Wyeast 1272 - American Ale IITM"
```
Comment (sammdec) [inline data/shopwrecker_circus.json]: ™

**Comment by sammdec** [comment]
Thanks again I have left some notes if you have any questions at all shoot them over.

**Comment by AliSkinner** [comment]
Cool. Thanks for the notes! I've updated accordingly.

File: data/dead_metaphor.json [hunk 21]
```diff
@@ -0,0 +1,117 @@
...
+        "name": "Magnum",
+        "amount": {
+          "value": 25,
+          "unit": "grams"
+        },
+        "add": "start",
+        "attribute": "bitter"
+      }
+    ],
+    "yeast": "Wyeast 1056 - American AleT™"
```
Comment (sammdec) [inline data/dead_metaphor.json]: Looks like a rogue "T" :smile:

File: data/dogma.json [hunk 22]
```diff
@@ -0,0 +1,147 @@
+{
+  "name": "Dogma",
+  "tagline": "Revamped wee heavy. Luscious. Malty. Fruity.",
```
Comment (sammdec) [inline data/dogma.json]: Would you mind making Wee & Heavy capital letters

**Comment by sammdec** [comment]
Just found a last couple of pinickity ones but apart from that looks ace :+1:

**Comment by sammdec** [comment]
Nailed it lgtm :+1:

**Comment by AliSkinner** [comment]
Ha! Sorry about that! Prob could have done them yourself in that time! Better quality coming in subsequent PRs!

**Comment by sammdec** [comment]
Not a problem at all it was a huge help so thanks very much! It took me a while to find a rhythm to it.

**Pull request merged**